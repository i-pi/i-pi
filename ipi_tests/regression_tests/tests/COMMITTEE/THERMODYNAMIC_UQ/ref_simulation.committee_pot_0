 #EXTRAS(committee_pot)# Step:           0 
 -2.04667050e+00  -2.04134110e+00
 #EXTRAS(committee_pot)# Step:           2 
 -2.41165670e+00  -2.41148881e+00
 #EXTRAS(committee_pot)# Step:           4 
 -2.89213351e+00  -2.89696109e+00
 #EXTRAS(committee_pot)# Step:           6 
 -2.85426510e+00  -2.85622969e+00
 #EXTRAS(committee_pot)# Step:           8 
 -2.47492091e+00  -2.46903886e+00
 #EXTRAS(committee_pot)# Step:          10 
 -2.49231870e+00  -2.48586641e+00
