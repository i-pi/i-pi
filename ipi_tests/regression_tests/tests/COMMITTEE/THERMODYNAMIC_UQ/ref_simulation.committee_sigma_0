 #EXTRAS(committee_uncertainty)# Step:           0  Bead:      0  
0.0037684520692876948
 #EXTRAS(committee_uncertainty)# Step:           2  Bead:      0  
0.00011871998408644411
 #EXTRAS(committee_uncertainty)# Step:           4  Bead:      0  
0.0034136092022918067
 #EXTRAS(committee_uncertainty)# Step:           6  Bead:      0  
0.0013891718026629338
 #EXTRAS(committee_uncertainty)# Step:           8  Bead:      0  
0.004159236446641155
 #EXTRAS(committee_uncertainty)# Step:          10  Bead:      0  
0.004562460384241514
