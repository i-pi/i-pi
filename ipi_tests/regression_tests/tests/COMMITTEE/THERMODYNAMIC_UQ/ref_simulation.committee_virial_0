 #EXTRAS(committee_virial)# Step:           0  Bead:      0  
    -0.81755615          -0.68559468          -0.07289286          -0.67048751          -0.87989917           4.77220104          -0.08631796          -0.23878892          -1.39727383
    -0.63184142          -0.69146025          -0.06919554          -0.67048751          -0.69504091           4.76776664          -0.08631796          -0.23878892          -1.21531437
 #EXTRAS(committee_virial)# Step:           2  Bead:      0  
    -1.55336750          -0.63634642           0.05347585          -0.61886016          -1.32719960           4.69956497           0.03564453          -0.31478512          -2.01597320
    -1.39300321          -0.64318135           0.05889872          -0.61886016          -1.16312628           4.69393378           0.03564453          -0.31478512          -1.86041964
 #EXTRAS(committee_virial)# Step:           4  Bead:      0  
    -2.27151879          -0.26774678           0.10877754          -0.25879253          -1.89848747           4.75250691           0.08996495          -0.27564736          -2.42433568
    -2.13552024          -0.27140894           0.11456444          -0.25879253          -1.75861680           4.74864099           0.08996495          -0.27564736          -2.29068620
 #EXTRAS(committee_virial)# Step:           6  Bead:      0  
    -1.48290621           0.30255193           0.09017584           0.29291020          -1.17006341           4.86377023           0.07143280          -0.18299097          -1.22366663
    -1.33300934           0.30607232           0.09605608           0.29291020          -1.01690720           4.86127117           0.07143280          -0.18299097          -1.07172652
 #EXTRAS(committee_virial)# Step:           8  Bead:      0  
     0.05524749           0.77964145          -0.01018797           0.75081025           0.05498487           5.01343016          -0.02384945          -0.05295096           0.66262336
     0.24491368           0.79078946          -0.00629817           0.75081025           0.24099876           5.01326218          -0.02384945          -0.05295096           0.85795972
 #EXTRAS(committee_virial)# Step:          10  Bead:      0  
     0.21328886           0.83776565          -0.13563500           0.80922800          -0.07800695           5.11938554          -0.14398534           0.03700369           0.96675199
     0.40610706           0.84864482          -0.13378192           0.80922800           0.10563437           5.12239271          -0.14398534           0.03700369           1.16954386
