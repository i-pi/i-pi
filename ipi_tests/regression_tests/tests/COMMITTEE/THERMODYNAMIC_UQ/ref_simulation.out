# column   1     --> step : The current simulation time step.
# column   2     --> time{picosecond} : The elapsed simulation time.
# column   3     --> conserved : The value of the conserved energy quantity per bead.
# column   4     --> temperature{kelvin} : The current temperature, as obtained from the MD kinetic energy.
# column   5     --> potential : The physical system potential energy.
# column   6     --> pressure_md{bar} : The pressure of the (extended) classical system.
# column   7     --> volume : The volume of the cell box.
    0.00000000e+00     0.00000000e+00    -1.09303818e+00     3.06943821e+02    -2.04400580e+00    -2.08913607e+03     4.37369880e+04   
    2.00000000e+00     1.00000000e-03    -1.10502261e+00     4.15313319e+02    -2.41157276e+00    -4.71489111e+03     4.37356814e+04   
    4.00000000e+00     2.00000000e-03    -1.11605581e+00     5.44332769e+02    -2.89454730e+00    -6.82080867e+03     4.37302571e+04   
    6.00000000e+00     3.00000000e-03    -1.11447080e+00     5.07914391e+02    -2.85524739e+00    -1.17812942e+03     4.37136394e+04   
    8.00000000e+00     4.00000000e-03    -1.10163975e+00     3.71799751e+02    -2.47197989e+00     7.50646938e+03     4.36953063e+04   
    1.00000000e+01     5.00000000e-03    -1.10059605e+00     3.70973913e+02    -2.48909256e+00     8.24566330e+03     4.36818932e+04   
